contrib/tlp/patches/01-tlp-func-base-in.patch

```diff
@@ -0,0 +1,26 @@
+--- tlp-func-base.in
```

Comment:
support for `flock -w` is coming in chimerautils 14.0

Reply:
Oh! niceee

btw when?